R/zzz_deprecated.R

```diff
@@ -0,0 +1,26 @@
+#' Deprecated functions
+#'
+#' @param ... Arguments to the deprecated function.
+#'
+#' @details
+#' - `pkAnalysesAsDataFrame` is now [`pkAnalysesToDataFrame`].
+#' - `populationAsDataFrame` is now [`populationToDataFrame`].
+#'
+#' @rdname ospsuite_deprecated
+#' @name ospsuite_deprecated
+NULL
+
+
+#' @rdname ospsuite_deprecated
+#' @export
+pkAnalysesAsDataFrame <- function(...) {
+  .Deprecated("pkAnalysesAsDataFrame")
+  pkAnalysesToDataFrame(...)
+}
+
+#' @rdname ospsuite_deprecated
+#' @export
+populationAsDataFrame <- function(...) {
```

Comment:
Oh that's awesome. It's using the details from the comments at the top of the file?

Reply:
Yes, and the docs just redirect the user to the docs for the primary function.

So:
- if someone uses `populationAsDataFrame(...)`, it will work but will issue a deprecation warning
- if someone searches for `?populationAsDataFrame`, it will take them to `ospsuite_depcreated` docs that detail which function they should instead be using